lib/Elastica/Cluster.php

```diff
@@ -73,14 +73,7 @@ public function getResponse()
      */
     public function getIndexNames()
     {
-        $metaData = $this->_data['metadata']['indices'];
-
-        $indices = array();
-        foreach ($metaData as $key => $value) {
-            $indices[] = $key;
-        }
-
-        return $indices;
+        return array_keys($this->_data['metadata']['indices']);
```

Comment:
Yep, looks good. That is what I was referring to: https://codecov.io/gh/ruflin/Elastica/src/bc7072afc02957035bd1c579e5dbd74e553950e4/lib/Elastica/Cluster.php#L74